agt_setup/__main__.py

```
@@ -0,0 +1,60 @@
+import os
+from pathlib import Path
+
+import typer
+from dotenv import set_key
+from neo4j import GraphDatabase
+
+app = typer.Typer(add_completion=False)
+
+
+from neo4j.exceptions import ServiceUnavailable, AuthError
```

Ruff (0.11.9)

Module level import not at top of file

(E402)